chia/util/chia_logging.py

```diff
@@ -1,12 +1,25 @@
 import logging
 from pathlib import Path
-from typing import Dict
+from typing import Any, Dict
 
 import colorlog
 from concurrent_log_handler import ConcurrentRotatingFileHandler
 from logging.handlers import SysLogHandler
 
+from chia.cmds.init_funcs import chia_full_version_str
 from chia.util.path import path_from_root
+from chia.util.default_root import DEFAULT_ROOT_PATH
+
+
+def get_beta_logging_config() -> Dict[str, Any]:
+    return {
+        "log_filename": f"{chia_full_version_str()}/chia-blockchain/beta.log",
```

Comment:
filename = version/chia-blockchain/beta.log ?
if i got it correct chia-blockchain seems redundant here

Reply:
This is just to have separate folders for normal logs vs plotting, see the structure here: 
![](https://user-images.githubusercontent.com/35775977/179071608-7208872a-bc8e-45c7-8e16-ae4b9960e4f1.png)